almalinux/9/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM almalinux:9
+MAINTAINER Oleg Chaplashkin <ochaplashkin@tarantool.org>
+
+# Enable extra tools
+RUN dnf -y install wget yum-utils
+
+# Enable extra repositories
+RUN dnf -y install epel-release
```

Comment:
Due to Docker architecture, this step as a separate one doesn't make big sense. One of the best practices for creating images states:

Try to reduce the number of layers in your image by minimizing the number of separate `RUN` commands in your Dockerfile. You can do this by consolidating multiple commands into a single `RUN` line and using your shell's mechanisms to combine them together.

Reply:
Fixed.